packages/ag-charts-community/src/chart/series/cartesian/lineUtil.ts

```diff
@@ -243,14 +237,14 @@ function pairCategoryData(
         if (diff.removed.indexOf(next.xValue) >= 0) {
             resultPoint = {
                 marker: 'out',
-                moveTo: next.point.moveTo,
+                moveTo: next.point.moveTo ? 'yes' : 'no',
```

Comment:
It might be nicer if the `moveTo` type was `boolean | 'in' | 'out'`, then you wouldn't need to do these ternaries.

Reply:
I had that originally, Alex asked me to change it to this :-P